code/game/machinery/washing_machine.dm

```diff
@@ -349,7 +349,8 @@ GLOBAL_LIST_INIT(dye_registry, list(
 		update_icon()
 
 /obj/machinery/washing_machine/deconstruct(disassembled = TRUE)
-	new /obj/item/stack/sheet/metal(drop_location(), 2)
+	if (!(flags_1 & HOLOGRAM_1))
+		new /obj/item/stack/sheet/metal(drop_location(), 2)
```

Comment:
Doesn't the holodeck already apply the NODECONSTRUCT flag that we could hook in here instead?

Reply:
That seems like a better solution for machinery/structures, done